frontend/class-twitter.php

```diff
@@ -375,15 +374,19 @@ private function gallery_images_output() {
 	 * @return bool
 	 */
 	private function taxonomy_image_output() {
+		$img = false;
 		foreach ( array( 'twitter-image', 'opengraph-image' ) as $tag ) {
 			$img = WPSEO_Taxonomy_Meta::get_meta_without_term( $tag );
-			if ( $img !== '' ) {
-				$this->image_output( $img );
-
-				return true;
+			if ( $img && $img !== '' ) {
+				break;
 			}
 		}
-
+		// Filter: 'wpseo_twitter_taxonomy_image' - Allow changing the Twitter taxonomy image.
```

Comment:
Can you use this way of documenting the filter:
```
/**
  * Filter: wpseo_twitter_taxonomy_image - Allow developers to set a image for the taxonomy.
  *
  * @api bool False by default, for not having a taxonomy image
  * 
  * @returns false|string A string when an image should be outputted. False if not.
  */
```

We are using this notation, because this allows us to generate proper documentation.